typings/config.ts

```diff
@@ -1,3 +1,5 @@
+import { Config } from "prettier";
```

Comment:
Unused import

Reply:
> You also need to resolve the merge conficts

i believe i can do that, however I am pretty sure they were caused by the PR before this

Reply:
removed